tests/NuGet.Services.AzureSearch.Tests/BaseDocumentBuilderFacts.cs

```diff
@@ -160,6 +161,83 @@ public void AllowNullDependencyGroups()
 
                 Assert.Null(full.FlattenedDependencies);
             }
+
+            [Fact]
+            public void IfLeafHasIconFile_LinksToFlatContainer()
+            {
+                var leaf = new PackageDetailsCatalogLeaf
+                {
+                    PackageId = Data.PackageId,
+                    PackageVersion = Data.NormalizedVersion,
+                    IconFile = "iconFile",
+                    IconUrl = "iconUrl"
+                };
+
+                var full = new HijackDocument.Full();
+
+                Target.PopulateMetadata(full, Data.NormalizedVersion, leaf);
+
+                Assert.Equal(
+                    $"{Data.FlatContainerBaseUrl}{Data.FlatContainerContainerName}/{Data.PackageId.ToLowerInvariant()}/{Data.NormalizedVersion.ToLowerInvariant()}/icon",
```

Comment:
nit: all of the inputs for the expected URL are in `Data`. You can maybe make an `FlatContainerIconUrl` property (or better name) to define it once for both assertions in your change set

Reply:
done!